x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/i18n_texts.ts

```diff
@@ -11,6 +11,13 @@ export const i18nTexts = {
     shrinkLabel: i18n.translate('xpack.indexLifecycleMgmt.shrink.indexFieldLabel', {
       defaultMessage: 'Shrink index',
     }),
+    rolloverOffsetsHotPhaseTiming: i18n.translate(
+      'xpack.indexLifecycleMgmt.rollover.rolloverOffsetsPhaseTimingDescription',
+      {
+        defaultMessage:
+          'When rollover is active, data age is only considered after rollover has occurred which adds variation to time in hot phase.',
+      }
```

Comment:
I think I'd just re-use the previous suggestion, the qualifier isn't necessary. 

```suggestion
          'How long it takes to reach the rollover criteria in the hot phase can vary. 
Data moves to the next phase when the time since rollover reaches the minimum age.',
      }
```